packages/@jsii/runtime/lib/sync-stdio.ts

```diff
@@ -53,7 +51,8 @@ export class SyncStdio {
         if (e.code !== 'EAGAIN') {
           throw e;
         }
-        sleep(50 /*ms*/);
+        // We'll retry immediately, and possibly thrash the CPU until the buffer was drained. We
```

Comment:
How many time will we retry? "thrash the cpu" sounds scary, should we even retry?

Reply:
We most Rettel until it succeeds... if we don't we crash. And apparently we retry for less than 5ms apiece 🙃